spec/models/professional_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.shared_examples "invalid professional" do |args|
+  subject { FactoryBot.build(:professional, args[:field]=>args[:value]) }
+
+  it { is_expected.to be_invalid }
+end
+
+RSpec.describe Professional, type: :model do
+  describe  'validation' do
+    context "presence with nil" do
```

Comment:
find a better naming
context should start with `when xxx` - https://www.betterspecs.org/#contexts